Stargate v2 readme updates (#2055)
stargate/stargate: apis/sgv2-docsapi/CONFIGURATION.md

The complete list of Quarkus available properties can be found on [All configuration options](https://quarkus.io/guides/all-config) page.

> However, here are some Stargate-relevant property groups that are important for correct setup of the service:

Reviewer: ```suggestion
Here are some Stargate-relevant property groups that are necessary for correct service setup:
```

Author: making this change in next commit
